Drop-In/src/main/java/com/braintreepayments/api/dropin/DropInRequest.java

```diff
@@ -182,6 +184,29 @@ public DropInRequest vaultManager(boolean vaultManager) {
         return this;
     }
 
+    /**
+     * @param defaultVaultValue {@code true} defaults to vaulting card info. {@code false} defaults to not vaulting card info.
+     * If Save Card CheckBox is shown and default vault value is true, CheckBox will appear pre-checked.
+     * If Save Card CheckBox is shown and default vault value is false, CheckBox will appear un-checked.
+     * If Save Card CheckBox is not shown and default vault value is false, card never vaults.
+     * If Save Card CheckBox is not shown and default vault value is true, card always vaults.
+     *
+     * This value is {@code true} by default.
+     */
+    public DropInRequest defaultVaultSetting(boolean defaultVaultValue) {
+        mDefaultVaultSetting = defaultVaultValue;
+        return this;
+    }
+
+    /**
+     * @param showCheckBox {@code true} shows Save Card CheckBox to allow user to choose whether or not to save/vault their card info.
```

Comment:
```
@param showCheckBox {@code true} shows save card CheckBox to allow user to choose whether or not to vault their card.
```

I think we can choose one of the following: `save/vault`. I think `vault` makes sense since it is the [official name](https://articles.braintreepayments.com/control-panel/vault/overview)

Reply:
👍